src/map/script.cpp

```diff
@@ -19928,6 +19937,7 @@ static int buildin_instance_warpall_sub(struct block_list *bl, va_list ap)
 	owner_id = instance_data[instance_id].owner_id;
 	switch(instance_data[instance_id].mode) {
 		case IM_NONE:
+		case IM_MAX:
```

Comment:
`IM_MAX` is afaik not a real instance mode. So it should fail in this case. Personally I'd just remove this case and leave it as it is.